index.md

```diff
@@ -140,7 +146,50 @@ In its simplest form, the `schema` requires you to provide a `name: value` pair
 </script>
 ```
 
-## Component Requirements
+### Prop: preventModelCleanupOnSchemaChange
+
+`SchemaForm` by default parses and clears the output whenever the `schema` property changes.
```

Comment:
```suggestion
By default `SchemaForm` cleans up the value output of properties that are no longer present inside `schema` every time `schema` changes.
```

This is explained nicely below but I think we might be more precise here for those that just end up skimming the 1st paragraph.